lib/services/http-service.js

```diff
@@ -142,6 +144,17 @@ function httpServiceFactory(
             });
         }
 
+        //Swagger support
+        var swaggerConfig = {
+            appRoot: path.resolve(__dirname, '../../'),
+            configDir: path.resolve(__dirname, '../../swagger_config'),
+            swagger: path.resolve(__dirname, '../../static/monorail-2.0.yaml'),
```

Comment:
@WangWinson, @jlongever's apidoc solution is a way of generating a swagger.json from doc comments.  Since we are actually installing swagger middleware now, swagger.json is generated from swagger.yaml by the middleware itself and thus the apidoc method is no longer needed.